src/ServerTelemetryChannel/SamplingTelemetryProcessor.cs

```diff
@@ -139,6 +153,19 @@ public string IncludedTypes
         /// </summary>
         private ITelemetryProcessor Next { get; set; }
 
+        /// <summary>
+        /// Gets or sets the next TelemetryProcessor to call in the chain if the ITelemetry item passed in is /not/ sampled.
+        /// 
+        /// <remarks>
+        /// Sampling telemetry processor is used standalone (can be added to the Application Insights config file
+        /// directly) as well as in a combination within the processing chain sandwiched between the 
+        /// AdaptiveSamplingTelemetryProcessor and the SamplingPercentageEstimatorTelemetryProcessor. We cannot count
+        /// unsampled items in this class's processor routine towards adaptive sampling samples, so we need to bypass
+        /// those unsampled items passed the estimator to be accurate. See issue (blah) for more details.
```

Comment:
?

Reply:
Removing the <remark> part of this comment altogether. This PR and the issue will serve as the documentation.